### PR TITLE
Fix pluralization of ammo.

### PR DESCRIPTION
**Pull request by actual-nh**
#### Summary

SUMMARY: None

#### Purpose of change

A number of the ammo entries had incorrect pluralization, such as
"##mm FMJ" being pluralized as "##mm FMJs" (when FMJ is essentially
acting as an adjective) or "buckshot" to "buckshots" when each round
has multiple pieces of buckshot.

#### Describe the solution

Ran a substitution of "str" to "str_sp" in data/json/items/ammo, then manually corrected the results. Also manually went over data/json/items/ammo.json.

#### Testing

JSON formatter run.

#### Additional context

Based off of #46628 by @BrettDong (including the substitution used), who said it was fine for me to make this one BTW. Hopefully will, as intended in that (closed) PR, reduce translator workload at least a bit.

### CODE REVIEW

**Comment by actual-nh** [comment]
@ToxiClay - if you (or someone else doing copyediting, of course) would like to check over my choices? Language is inevitably subjective...

File: data/json/items/ammo.json [hunk 1]
```diff
@@ -293,7 +293,7 @@
     "category": "spare_parts",
     "price": 2000,
     "price_postapoc": 100,
-    "name": { "str": "copper wire" },
+    "name": { "str_sp": "copper wire" },
```
Comment (ToxiClay) [inline data/json/items/ammo.json:296]:
```suggestion
    "name": { "str_sp": "lengths of copper wire" },
```

Comment (ToxiClay) [inline data/json/items/ammo.json:296]:
```suggestion
    "name": { "str": "copper wire" },
    "name": { "str_pl": "lengths of copper wire" },
```

File: data/json/items/ammo.json [hunk 2]
```diff
@@ -467,7 +467,7 @@
     "id": "bb",
     "price": 500,
     "price_postapoc": 50,
-    "name": { "str": "BB" },
+    "name": { "str_sp": "BBs" },
```
Comment (ToxiClay) [inline data/json/items/ammo.json:470]:
```suggestion
    "name": { "str": "BB" },
    "name": { "str_pl": "BBs" },
```

Remember, str_sp is used for both singular *and* plural; "1 BBs" is just as wrong as "50 BB."

Comment (actual-nh) [reply data/json/items/ammo.json:470]: Then "box of BBs" vs "boxes of BBs"?

File: data/json/items/ammo.json [hunk 3]
```diff
@@ -524,7 +524,7 @@
     "id": "120mm_HEAT",
     "price": 999999,
     "price_postapoc": 1000,
-    "name": { "str": "120mm HEAT" },
+    "name": { "str_sp": "120mm HEAT" },
```
Comment (ToxiClay) [inline data/json/items/ammo.json:527]:
```suggestion
    "name": { "str": "120mm HEAT round" },
```

"HEAT rounds" is a good pluralization.

File: data/json/items/ammo.json [hunk 4]
```diff
@@ -804,7 +804,7 @@
     "category": "spare_parts",
     "price": 2000,
     "price_postapoc": 100,
-    "name": { "str": "medical tape" },
+    "name": { "str": "medical tape", "str_pl": "medical tape rolls" },
```
Comment (ToxiClay) [inline data/json/items/ammo.json:807]:
```suggestion
    "name": { "str": "medical tape", "str_pl": "rolls of medical tape" },
```
Check to see if you can concatenate like this without the game exploding. I don't think you can.

File: data/json/items/ammo.json [hunk 5]
```diff
@@ -258,7 +258,7 @@
     "category": "spare_parts",
     "price": 2000,
     "price_postapoc": 250,
-    "name": { "str": "duct tape" },
+    "name": { "str": "duct tape", "str_pl": "duct tape rolls" },
```
Comment (ToxiClay) [inline data/json/items/ammo.json:261]:
```suggestion
    "name": { "str": "duct tape", "str_pl": "rolls of duct tape" },
```
Check to see if you can concatenate `name` like this.

File: data/json/items/ammo.json [hunk 6]
```diff
@@ -141,7 +141,7 @@
     "category": "spare_parts",
     "price": 648,
     "price_postapoc": 10,
-    "name": { "str": "aluminum foil" },
+    "name": { "str_sp": "aluminum foil" },
```
Comment (ToxiClay) [inline data/json/items/ammo.json:144]:
```suggestion
    "name": { "str_sp": "pieces of aluminum foil" },
```

File: data/json/items/ammo.json (hunk 2 above)
Comment (ToxiClay) [reply data/json/items/ammo.json:470]: Oh, wait, yes, apparently. Its description is a box of BBs, but ... is this a single BB in game? I can't tell.

File: data/json/items/ammo.json [hunk 7]
```diff
@@ -206,7 +206,7 @@
     "category": "spare_parts",
     "price": 0,
     "price_postapoc": 25,
-    "name": { "str": "plant fiber" },
+    "name": { "str_sp": "plant fiber" },
```
Comment (ToxiClay) [inline data/json/items/ammo.json:209]:
```suggestion
    "name": { "str": "plant fiber" },
    "name": { "str_pl": "pieces of plant fiber" },
```

File: data/json/items/ammo.json [hunk 8]
```diff
@@ -563,7 +563,7 @@
     "category": "chems",
     "price": 500,
     "price_postapoc": 2000,
-    "name": { "str": "mixed smokeless gunpowder" },
+    "name": { "str_sp": "mixed smokeless gunpowder" },
```
Comment (ToxiClay) [inline data/json/items/ammo.json:566]: Part of me wants to make the plural "units of smokeless gunpowder," but I don't know if "units" would be too...ambiguous?

File: data/json/items/ammo.json (hunk 6 above)
Comment (actual-nh) [reply data/json/items/ammo.json:144]:
```suggestion
    "name": { "str": "piece of aluminum foil", "str_pl": "pieces of aluminum foil" },
```

File: data/json/items/ammo.json [hunk 9]
```diff
@@ -788,7 +788,7 @@
     "category": "spare_parts",
     "price": 1000,
     "price_postapoc": 250,
-    "name": { "str": "copper" },
+    "name": { "str_sp": "copper" },
```
Comment (ToxiClay) [inline data/json/items/ammo.json:791]:
```suggestion
    "name": { "str": "copper" },
    "name": { "str_pl": "scraps of copper" },
```

File: data/json/items/ammo.json [hunk 10]
```diff
@@ -868,7 +868,7 @@
     "category": "spare_parts",
     "price": 200,
     "price_postapoc": 100,
-    "name": { "str": "coal" },
+    "name": { "str": "coal", "str_pl": "coal lumps" },
```
Comment (ToxiClay) [inline data/json/items/ammo.json:871]:
```suggestion
    "name": { "str": "coal", "str_pl": "lumps of coal" },
```

File: data/json/items/ammo.json [hunk 11]
```diff
@@ -884,7 +884,7 @@
     "type": "AMMO",
     "id": "albuterol",
     "category": "drugs",
-    "name": { "str": "albuterol" },
+    "name": { "str_sp": "albuterol" },
```
Comment (ToxiClay) [inline data/json/items/ammo.json:887]:
```suggestion
    "name": { "str": "charge of albuterol" },
    "name": { "str_sp": "charges of albuterol" },
```

File: data/json/items/ammo/10mm.json [hunk 12]
```diff
@@ -2,7 +2,7 @@
   {
     "id": "10mm_fmj",
     "type": "AMMO",
-    "name": { "str": "10mm Auto FMJ" },
+    "name": { "str_sp": "10mm Auto FMJ" },
```
Comment (ToxiClay) [inline data/json/items/ammo/10mm.json:5]: Where do these strings appear in-game? If I were to suggest "X rounds of 10mm Auto FMJ," and similarly for the other ammo types, would it break the screen too badly?

File: data/json/items/ammo.json (hunk 6 above)
Comment (ToxiClay) [reply data/json/items/ammo.json:144]: Did you check in your game to see if the game accepts that? What do the maintainers think of this style?

Comment (actual-nh) [reply data/json/items/ammo.json:144]: Is your concern re size on the screen, or what? Existing ones have things like `{ "str": "chunk of sulfur", "str_pl": "chunks of sulfur" }`. I'll check it when I can, but I'm going to be occupied with teaching this coming week.

File: data/json/items/ammo.json (hunk 5 above)
Comment (actual-nh) [reply data/json/items/ammo.json:261]: Are you referring to adding "rolls" for a plural? In which position?

File: data/json/items/ammo.json (hunk 8 above)
Comment (actual-nh) [reply data/json/items/ammo.json:566]: From the "count" below, it's multiple units in the first place, it looks like.

File: data/json/items/ammo.json (hunk 6 above)
Comment (ToxiClay) [reply data/json/items/ammo.json:144]: Oh, if existing items in the game have the "name" members all on the same line, and the game works, that's fine. My concern was whether the game would know what to do with it.

File: data/json/items/ammo.json (hunk 5 above)
Comment (ToxiClay) [reply data/json/items/ammo.json:261]: Yeah, switching "duct tape rolls" for "rolls of duct tape."

File: data/json/items/ammo/10mm.json (hunk 12 above)
Comment (actual-nh) [reply data/json/items/ammo/10mm.json:5]: The context is mixed. It can be talking about what ammunition a gun takes, in which case "round of"/"rounds of" is just repeating the obvious. Or it can be talking about something in a container, in which case singular vs plural does make a difference. (And there are probably other cases that I'm not thinking of - neither @BrettDong nor @Qrox were quite sure of all the different places they could be used; see the earlier PR I referenced.)

Comment (ToxiClay) [reply data/json/items/ammo/10mm.json:5]: Oof, yeah. And I am *loathe* to suggest adding another field (and the relevant C++ support) to handle separating those cases.

File: data/json/items/ammo.json (hunk 6 above)
Comment (actual-nh) [reply data/json/items/ammo.json:144]: Oh. Yes, that's how one is supposed to do it (it may be put on the next line in terms of a newline (as opposed to another "name":), but that the JSON formatter can take care of).

File: data/json/items/ammo.json (hunk 3 above)
Comment (actual-nh) [reply data/json/items/ammo.json:527]: True. I'm not sure if it's worth it lengthening it in the "what does this gun use" context, though. Something to be discussed.

File: data/json/items/ammo/10mm.json (hunk 12 above)
Comment (actual-nh) [reply data/json/items/ammo/10mm.json:5]: What BrettDong was wanting to do was convert as much to str_sp as possible to alleviate burden on the translators (even worse than excess entries were things like "9mm NATO, reloaded" becoming "9mm NATO, reloadeds" and some poor person being asked to translate that...).

Comment (ToxiClay) [reply data/json/items/ammo/10mm.json:5]: Ahh. OK, if the primary goal is to help the translators, then I think it's OK to accept a little bit of "maybe this isn't *super* grammatical" in service to that.

File: data/json/items/ammo.json (hunk 2 above)
Comment (actual-nh) [reply data/json/items/ammo.json:470]: The "count" field says it's 500 units of ammunition, as far as I can tell.

File: data/json/items/ammo.json (hunk 9 above)
Comment (actual-nh) [reply data/json/items/ammo.json:791]: The description and "count" field seem to be indicating that it's already multiple scraps. Your suggestion, BTW, would be done as:
```suggestion
    "name": { "str": "copper", "str_pl": "scraps of copper" },
```

File: data/json/items/ammo/10mm.json (hunk 12 above)
Comment (actual-nh) [reply data/json/items/ammo/10mm.json:5]: Of course, the description below appears to be contradictory with the "count" field, although that the "stack_size" is identical may be an explanation (I _think_ that means you can work with one at a time, but I'm not sure).

File: data/json/items/ammo/20x66mm.json [hunk 13]
```diff
@@ -102,7 +102,7 @@
     "id": "20x66_inc",
     "copy-from": "20x66_shot",
     "type": "AMMO",
-    "name": { "str": "20x66mm incendiary", "str_pl": "20x66mm incendiaries" },
+    "name": { "str_sp": "20x66mm incendiary" },
```
Comment (actual-nh) [inline data/json/items/ammo/20x66mm.json:105]: The problem here being that "20x66mm incendiaries" means more than one different type of incendiary round (incendiary is an adjective, not a noun....).

File: data/json/items/ammo.json (hunk 9 above)
Comment (ToxiClay) [reply data/json/items/ammo.json:791]: Sure, but when you pop open your inventory and you see `5 copper` on the ground, it's not entirely clear what that means. 5 copper pieces, like this is D&D? Sheets of copper?

File: data/json/items/ammo/10mm.json (hunk 12 above)
Comment (ToxiClay) [reply data/json/items/ammo/10mm.json:5]: Probably that last one; there's a lot of duct tape and scuff marks in the game's code at the moment. :)

File: data/json/items/ammo/20x66mm.json (hunk 13 above)
Comment (ToxiClay) [reply data/json/items/ammo/20x66mm.json:105]: *Is* there more than one type of _20x66_ incendiary, though?

Comment (ToxiClay) [reply data/json/items/ammo/20x66mm.json:105]: If someone tosses you an incendiary grenade and you ask what it is, "An incendiary" is a grammatically-acceptable response, since "grenade" or "device" is implied by context.

File: data/json/items/ammo/10mm.json (hunk 12 above)
Comment (Qrox) [reply data/json/items/ammo/10mm.json:5]: According to `doc/JSON_INFO.md`,

```c++
"count" : 25,         // Number of rounds that spawn together
"stack_size" : 50,    // (Optional) How many rounds are in the above-defined volume. If omitted, is the same as 'count'
```

So a charge of ammo is a round of ammo. The description of BBs is a bit confusing here.

**Comment by Qrox** [comment]
Items of the `AMMO` and `COMESTIBLE` types only had plural names after I migrated item names to `class translation`. And the only change since then (that I'm aware of) that uses the plural names is the recent change to describe contents in container names. So it might be safe to assume that container names are the only place where plural is used for `AMMO` items.

Adding modifiers such as `pieces of` or `rolls of` to item names may seem a bit wordy and sometimes incorrect though, as a charge of item is only an arbitrary minimal amount one can work with, which is not necessarily in some defined shape or form. I would just accept the imperfect grammar and use the same singular and plural form if a concise, correct pluralization is not possible.

**Comment by CountAlex** [comment]
I would really appreciate if this sort of adjusting names for so many entries would be more or less final. I've done ammo translation once when proof-reading, second time when it moved to singular-plural naming (and that's 4x tedious part of a job to do as there were 4 forms of every entry), and it seems I (and other translators, for every language) will have to do that again, which I guess is fine. Still, please consider lowering workload on translators as Transifex is a bit painfully tedious tool to use when it comes to that many entries to translate.

**Comment by actual-nh** [comment]
> I would really appreciate if this sort of adjusting names for so many entries would be more or less final. 

I understand and agree. A definite goal in this is to minimize the new strings to translate and maximize the removal of not-needed ones.

File: data/json/items/ammo.json (hunk 6 above)
Comment (actual-nh) [reply data/json/items/ammo.json:144]: I'd like to keep the original "str_sp", to reduce translator burden.

Comment (ToxiClay) [reply data/json/items/ammo.json:144]: Yeah. I'm OK with that, from below.

File: data/json/items/ammo/500.json [hunk 14]
```diff
@@ -26,7 +26,7 @@
     "id": "bp_500_Magnum",
     "copy-from": "500_Magnum",
     "type": "AMMO",
-    "name": { "str": ".500 S&W Magnum, black powder" },
+    "name": { "str": ".500 S&W Magnum, black powder", "str_pl": ".500 S&W Magnums, black powder" },
```
Comment (actual-nh) [inline data/json/items/ammo/500.json:29]:
```suggestion
    "name": { "str_sp": ".500 S&W Magnum, black powder" },
```
"Magnums" are only used in referring to guns firing the ammunition.

File: data/json/items/ammo/500.json [hunk 15]
```diff
@@ -40,7 +40,7 @@
     "id": "reloaded_500_Magnum",
     "copy-from": "500_Magnum",
     "type": "AMMO",
-    "name": { "str": ".500 S&W Magnum, reloaded" },
+    "name": { "str": ".500 S&W Magnum, reloaded", "str_pl": ".500 S&W Magnums, reloaded" },
```
Comment (actual-nh) [inline data/json/items/ammo/500.json:43]:
```suggestion
    "name": { "str_sp": ".500 S&W Magnum, reloaded" },
```

**Review by actual-nh — COMMENTED** [review]
I've done some double-checking, and some of the various names are only used in plural when talking about guns firing that type of ammunition (e.g., ".44 Magnums" means "guns firing .44 Magnums").

File: data/json/items/ammo/300.json [hunk 16]
```diff
@@ -26,7 +26,7 @@
     "id": "bp_300_winmag",
     "copy-from": "300_winmag",
     "type": "AMMO",
-    "name": { "str": ".300 Winchester Magnum, black powder" },
+    "name": { "str": ".300 Winchester Magnum, black powder", "str_pl": ".300 Winchester Magnums, black powder" },
```
Comment (actual-nh) [inline data/json/items/ammo/300.json:29]:
```suggestion
    "name": { "str_sp": ".300 Winchester Magnum, black powder" },
```

File: data/json/items/ammo/300.json [hunk 17]
```diff
@@ -40,7 +40,7 @@
     "id": "reloaded_300_winmag",
     "copy-from": "300_winmag",
     "type": "AMMO",
-    "name": { "str": ".300 Winchester Magnum, reloaded" },
+    "name": { "str": ".300 Winchester Magnum, reloaded", "str_pl": ".300 Winchester Magnums, reloaded" },
```
Comment (actual-nh) [inline data/json/items/ammo/300.json:43]:
```suggestion
    "name": { "str_sp": ".300 Winchester Magnum, reloaded" },
```

File: data/json/items/ammo.json [hunk 18]
```diff
@@ -467,7 +467,7 @@
     "id": "bb",
     "price": 500,
     "price_postapoc": 50,
-    "name": { "str": "BB" },
+    "name": { "str: "BB" },
```
Comment (actual-nh) [inline data/json/items/ammo.json:470]:
```suggestion
    "name": { "str": "BB" },
```

**Comment by actual-nh** [comment]
Travis build was cancelled for some reason?

File: data/json/items/ammo.json (hunk 2 above)
Comment (actual-nh) [reply data/json/items/ammo.json]: With the below correction from Qrox, I think just putting it back to "str": "BB" is preferable.

File: data/json/items/ammo.json (hunk 3 above)
Comment (actual-nh) [reply data/json/items/ammo.json:527]: I am inclined to not add the word.

File: data/json/items/ammo.json (hunk 8 above)
Comment (actual-nh) [reply data/json/items/ammo.json:566]: Well, it may not be multiple units, but I'm unsure exactly what a "unit" would be... for powders, really, it should be number of full containers, number of pounds/kg, or number of L/whatever.

File: data/json/items/ammo.json (hunk 9 above)
Comment (actual-nh) [inline data/json/items/ammo.json:791]:
```suggestion
    "name": { "str": "copper scrap" },
```

Comment (actual-nh) [reply data/json/items/ammo.json:791]: Below, I've suggested simply "str": "copper scrap".

File: data/json/items/ammo.json (hunk 11 above)
Comment (actual-nh) [reply data/json/items/ammo.json:887]: As per the discussion of translator burden, I am disinclined to expand the word.

File: data/json/items/ammo.json (hunk 9 above)
Comment (actual-nh) [reply data/json/items/ammo.json:791]: Maybe. See my next comment.

**Comment by actual-nh** [comment]
What's the best way to tell if a phrase is already being presented to the translators, NOT from the ammo files? Does .pot - if not compiled - have this information in a readable format, in which case running a modified update_pot.sh would work? I'm wondering if some duplication of translator effort could be avoided by using phrases that are used elsewhere in any event.

**Comment by BrettDong** [comment]
1. `lang/update_pot.sh` to generate `lang/po/cataclysm-dda.pot`
2. `grep -r 'phrase' lang/po/cataclysm-dda.pot`

File: data/json/items/ammo.json (hunk 5 above)
Comment (actual-nh) [inline data/json/items/ammo.json:261]:
```suggestion
    "name": { "str_sp": "duct tape" },
```

File: data/json/items/ammo.json [hunk 19]
```diff
@@ -293,7 +293,7 @@
     "category": "spare_parts",
     "price": 2000,
     "price_postapoc": 100,
-    "name": { "str": "copper wire" },
+    "name": { "str": "copper wire", "str_pl": "lengths of copper wire" },
```
Comment (actual-nh) [inline data/json/items/ammo.json:296]:
```suggestion
    "name": { "str": "copper wire" },
```

File: data/json/items/ammo.json [hunk 20]
```diff
@@ -788,7 +788,7 @@
     "category": "spare_parts",
     "price": 1000,
     "price_postapoc": 250,
-    "name": { "str": "copper" },
+    "name": { "str_sp": "copper" },
     "symbol": "=",
     "color": "yellow",
     "description": "Copper scraps.  Could be used to craft something, for example makeshift shotgun shells.",
```
Comment (actual-nh) [inline data/json/items/ammo.json:794]:
```suggestion
    "description": "Copper bits.  Could be used to craft something, for example makeshift shotgun shells.",
```
It takes 25 of these to make a "copper scrap", by requirements/materials.json.

File: data/json/items/ammo.json (hunk 4 above)
Comment (actual-nh) [inline data/json/items/ammo.json:807]:
```suggestion
    "name": { "str_sp": "medical tape" },
```
That you find 35 of them at the same time indicates a low likelihood of the description being right.

File: data/json/items/ammo.json (hunk 11 above)
Comment (actual-nh) [reply data/json/items/ammo.json:887]: 1 "charge"/dose of albuterol is actually 96 micrograms. (My spouse has asthma.)

File: data/json/items/ammo/300blk.json [hunk 21]
```diff
@@ -26,7 +26,7 @@
     "id": "300blk_ss",
     "copy-from": "300blk",
     "type": "AMMO",
-    "name": { "str": ".300 AAC Blackout subsonic" },
+    "name": { "str_sp": ".300 AAC Blackout subsonic" },
```
Comment (actual-nh) [inline data/json/items/ammo/300blk.json:29]:
```suggestion
    "name": { "str": ".300 AAC Blackout subsonic" },
```

File: data/json/items/ammo/50.json [hunk 22]
```diff
@@ -36,7 +36,7 @@
     "id": "50match",
     "copy-from": "50bmg",
     "type": "AMMO",
-    "name": { "str": ".50 BMG Match", "str_pl": ".50 BMG Matches" },
+    "name": { "str_sp": ".50 BMG Match" },
```
Comment (actual-nh) [inline data/json/items/ammo/50.json:39]: "Match" refers to what they're used in...

File: data/json/items/ammo.json [hunk 23]
```diff
@@ -206,7 +206,7 @@
     "category": "spare_parts",
     "price": 0,
     "price_postapoc": 25,
-    "name": { "str": "plant fiber" },
+    "name": { "str": "plant fiber", "str_pl": "pieces of plant fiber" },
```
Comment (actual-nh) [inline data/json/items/ammo.json:209]:
```suggestion
    "name": { "str": "plant fiber" },
```

**Review by actual-nh — COMMENTED** [review]
Further translator-friendly tweaks.

File: data/json/items/ammo.json [hunk 24]
```diff
@@ -996,7 +996,7 @@
     "category": "drugs",
     "price": 1000,
     "price_postapoc": 500,
-    "name": { "str": "nicotine liquid" },
+    "name": { "str_sp": "nicotine liquid" },
     "symbol": "=",
     "color": "yellow",
     "description": "A liquid that is made up of propylene glycol, vegetable glycerin, flavorings and nicotine.",
```
Comment (actual-nh) [inline data/json/items/ammo.json:1002]:
```suggestion
    "description": "A liquid that is made up of propylene glycol, vegetable glycerin, flavorings, and nicotine.",
```

File: data/json/items/ammo/20x66mm.json (hunk 13 above)
Comment (actual-nh) [inline data/json/items/ammo/20x66mm.json:105]:
```suggestion
    "name": { "str": "20x66mm incendiary", "str_pl": "20x66mm incendiaries" },
```

Comment (actual-nh) [reply data/json/items/ammo/20x66mm.json:105]: Point.

File: data/json/items/ammo.json (hunk 9 above)
Comment (actual-nh) [reply data/json/items/ammo.json:791]: Oh. And I've also changed the description from "copper scraps" to "copper bits", to avoid confusion with "scrap copper".

File: data/json/items/ammo/20x66mm.json (hunk 13 above)
Comment (actual-nh) [reply data/json/items/ammo/20x66mm.json]: Reverted to original.

**Comment by actual-nh** [comment]
Current status:
- Removes 229 phrases entirely
- Swaps out 13 for more grammatical ones (e.g., "shotgun slug, reloadeds" to "shotgun slugs, reloaded").
@BrettDong, @Qrox, @CountAlex, @ToxiClay, anyone else who'd care to comment - evaluations?

File: data/json/items/ammo/300blk.json [hunk 25]
```diff
@@ -37,7 +37,7 @@
     "id": "bp_300blk",
     "copy-from": "300blk",
     "type": "AMMO",
-    "name": { "str": ".300 AAC Blackout, black powder" },
+    "name": { "str": ".300 AAC Blackout, black powder", "str_pl": ".300 AAC Blackouts, black powder" },
```
Comment (Qrox) [inline data/json/items/ammo/300blk.json:40]:
```suggestion
    "name": { "str_sp": ".300 AAC Blackout, black powder" },
```
Or is there any difference that makes `Blackouts` acceptable here?

File: data/json/items/ammo/5x50.json [hunk 26]
```diff
@@ -38,7 +38,7 @@
     "id": "reloaded_5x50dart",
     "copy-from": "5x50dart",
     "type": "AMMO",
-    "name": { "str": "5x50mm flechette, reloaded" },
+    "name": { "str": "5x50mm flechette, reloaded", "str_pl": "5x50mm flechettes, reloaded" },
```
Comment (Qrox) [inline data/json/items/ammo/5x50.json:41]:
```suggestion
    "name": { "str_sp": "5x50mm flechette, reloaded" },
```
For consistency with other flechette rounds.

File: data/json/items/ammo/20x66mm.json [hunk 27]
```diff
@@ -102,7 +102,7 @@
     "id": "20x66_inc",
     "copy-from": "20x66_shot",
     "type": "AMMO",
-    "name": { "str_sp": "20x66mm incendiary" },
+    "name": { "str": "20x66mm incendiary", "str_pl": "20x66mm incendiaries" },
```
Comment (Qrox) [inline data/json/items/ammo/20x66mm.json]: Other incendiary rounds such as `7.62x51mm incendiary, reloaded` don't have a different plural, so it might be better to keep this change or change the other rounds to have a different plural for consistency.

File: data/json/items/ammo/5x50.json (hunk 26 above)
Comment (actual-nh) [reply data/json/items/ammo/5x50.json:41]: In the other flechette rounds, they actually have multiple flechettes in each round. This one only has one per round.

File: data/json/items/ammo/300blk.json (hunk 25 above)
Comment (actual-nh) [reply data/json/items/ammo/300blk.json:40]: I looked up the usage of the term - people do write (online, at least) `Blackouts` when they're talking about more than one `AAC Blackout` round. (They don't write "FMJs", however, and "Magnums" appears to be only used in talking about guns that fire them.)

File: data/json/items/ammo/20x66mm.json (hunk 27 above)
Comment (actual-nh) [reply data/json/items/ammo/20x66mm.json]: I keep going back and forth on this one.

File: data/json/items/ammo/20x66mm.json (hunk 13 above)
Comment (actual-nh) [reply data/json/items/ammo/20x66mm.json:105]: I just realized - these are used in a list (that includes something indicating rounds, 20x66mm).

File: data/json/items/ammo/20x66mm.json (hunk 27 above)
Comment (actual-nh) [reply data/json/items/ammo/20x66mm.json]: Went back to "str_sp" and "incendiary".

File: data/json/items/ammo/300blk.json (hunk 25 above)
Comment (actual-nh) [reply data/json/items/ammo/300blk.json:40]: In other words, "Blackout" is a noun, not an adjective (modifying "round" or "rounds").
